acai-ios/DetailViewControllers/BowlOrderDetailViewController.swift

```diff
@@ -57,8 +57,8 @@ class BowlOrderDetailViewController: UIViewController {
         // TODO: change to endpoint request data
         bowlItem = Acai.testBowl
         title = bowlItem.title
-        toppingOptions = bowlItem.toppingOptions
-        baseOptions = bowlItem.baseOptions
+        toppingOptions = bowlItem.options.filter({$0.type == .topping})
+        baseOptions = bowlItem.options.filter({$0.type == .base})
```

Comment:
GOOD JOB USING FILTER!!! 
add space after `{` and before `}` so something like:
`.filter({ $0.type == .base })`

Reply:
bump on this